tidb-cloud/v8.5-performance-benchmarking-with-sysbench.md

```diff
@@ -0,0 +1,366 @@
+
+- TiKV parameters:
+
+    ```yaml
+    [server]
+    grpc-concurrency = 6
+    ```
+
+- TiDB variables:
+
+    ```sql
+    set global tidb_enable_stmt_summary=off;
+    set global tidb_enable_collect_execution_info=off;
+    set global tidb_use_plan_baselines=off;
+    set global tidb_executor_concurrency=1;
+    ```
+
+For the `oltp_write_only` and `oltp_read_write` workload, use the follwing additional value of the parameters and variables:
+
+- TiKV parameters:
+
+    ```yaml
+    [server]
+    grpc-concurrency = 6
+    [raftstore]
+    apply-pool-size = 3
+    store-pool-size = 3
+    ```
+
+- TiDB variables:
+
+    ```sql
+    set global tidb_enable_stmt_summary=off;
+    set global tidb_enable_collect_execution_info=off;
+    set global tidb_txn_assertion_level=off;
+    set global tidb_guarantee_linearizability=off;
+    set global tidb_executor_concurrency=1;
+    ```
+
+### Benchmark executor
+
+The benchmark executor sends SQL queries to the TiDB cluster. In this test, its hardware configuration is as follows:
+
+- Machine type: Amazon EC2 (us-west-2)
+- Instance type: c6a.2xlarge
+- Sysbench version: sysbench 1.0.20 (using bundled LuaJIT 2.1.0-beta2)
+
+## Test steps
+
+This section introduces how to perform the Sysbench performance test step by step.
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Cloud Dedicated cluster that meets the [test environment](#tidb-cluster) requirements.
+
+    For more information, see [Create a TiDB Cloud Dedicated cluster](/tidb-cloud/create-tidb-cluster.md).
+
+2. On the benchmark executor, connect to the newly created cluster and create a database named `sbtest`.
+
+    To connect to the cluster, see [Connect to TiDB Cloud Dedicated via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md).
+
+    To create the `sbtest` database, execute the following SQL statement:
+
+    ```sql
+    CREATE DATABASE sbtest;
+    ```
+
+3. Load Sysbench data to the `sbtest` database.
+
+    - The test in this document is implemented based on [sysbench](https://github.com/akopytov/sysbench). To install sysbench, see [Building and installing from source](https://github.com/akopytov/sysbench#building-and-installing-from-source).
+
+    - Run the following `sysbench prepare` command to import 12 tables and 10,000,000 rows to the `sbtest` database. Replace `${HOST}`, `${PORT}`, `${THREAD}`, and `${PASSWORD}` with your actual values.
+
+      ```shell
+        sysbench \
+            --threads=${THREAD} \
+            --rand-type=uniform \
+            --db-driver=mysql \
+            --mysql-db=sbtest \
+            --mysql-host=${HOST} \
+            --mysql-port=${PORT} \
+            --mysql-user=root \
+            --tables=12 \
+            --table-size=10000000 \
+            --mysql-password=${PASSWORD} \
+            oltp_common prepare
+
+        for i in {1..12}; do
+        mysql -u root -h tidb-0 -P 4000 test -e "ANALYZE TABLE sbtest$i"
+        done
+      ```
```

Comment:
```suggestion
        ```shell
          sysbench \
              --threads=${THREAD} \
              --rand-type=uniform \
              --db-driver=mysql \
              --mysql-db=sbtest \
              --mysql-host=${HOST} \
              --mysql-port=${PORT} \
              --mysql-user=root \
              --tables=12 \
              --table-size=10000000 \
              --mysql-password=${PASSWORD} \
              oltp_common prepare

          for i in {1..12}; do
          mysql -u root -h tidb-0 -P 4000 test -e "ANALYZE TABLE sbtest$i"
          done
        ```
```